### PR TITLE
Add 01-authorization.md

### PR DESCRIPTION
**Pull request by alexisluque**

### CODE REVIEW

File: articles/quickstart/backend/django/01-authorization.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    'JWT_ISSUER': '${account.namespace}',
+    'JWT_AUTH_HEADER_PREFIX': 'Bearer',
+}
+```
+
+## Add a Django User
+
+You need to define a way to map the username from the `access_token` payload to the Django authentication system user.
+
+Create `user.py` file in your application's folder and define a function that maps the  `sub` field from the `access_token` to the username.
```
Comment (jeffreylees) [inline articles/quickstart/backend/django/01-authorization.md]: remove the second space before `sub`

File: articles/quickstart/backend/django/01-authorization.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+To configure scopes, click the Scopes section of the [Dashboard's APIs section](${manage_url}/#/apis) and configure the scopes you need.
+
+::: note
+This example uses the read:messages scopes.
+:::
+
+API endpoints can be configured to look for a particular `scope` in the `access_token`.
+
+Add the following methods to the `views.py` file to extract the granted scopes from the access_token.
```
Comment (jeffreylees) [inline articles/quickstart/backend/django/01-authorization.md]: When using "access_token" should probably be consistent in how it's notated. I'd stick with `access_token` 
 in code syntax when referring to _the_ token you're currently working with, and "access token" when discussing access tokens in general. But either way.

File: articles/quickstart/backend/django/01-authorization.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+```
+
+## Add URL Mappings
+
+In previous steps we added methods to the `views.py` file. We need to map those methods to URLs.
+
+Django has a [URL dispatcher](
+https://docs.djangoproject.com/en/1.11/topics/http/urls/) that lets you map URL patterns to views.
+
+Create the file `urls.py` in your application folder. Add the URLs patterns.
```
Comment (jeffreylees) [inline articles/quickstart/backend/django/01-authorization.md]: URL patterns

**Review by jeffreylees — COMMENTED** [review]
Just a couple of almost negligible copyedit suggestions on this one.

**Review by jeffreylees — APPROVED** [review]

**Review by aaguiarz — APPROVED** [review]

**Pull request merged**